### PR TITLE
Create devcontainer.json

### PR DESCRIPTION
**Pull request by Jordanwaslistening**
`"ghcr.io/devcontainers/features/aws-cli:1":` {}

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (Jordanwaslistening) [inline .devcontainer/devcontainer.json:1]: 👍

**Review by Jordanwaslistening — COMMENTED** [review]
👍

**Review by Jordanwaslistening — COMMENTED** [review]
### 

> **_`jjm`_**

**Comment by bamurtaugh** [comment]
Thanks again for your interest in contributing. As I mentioned in the other couple of PRs, going to close this one as I don't think it's intended for this repo.

**Pull request closed without merging**